src/main/java/uk/gov/hmcts/cmc/claimstore/models/ServiceAddress.java

```diff
@@ -0,0 +1,93 @@
+package uk.gov.hmcts.cmc.claimstore.models;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import org.hibernate.validator.constraints.NotBlank;
+
+import java.util.Objects;
+import javax.validation.constraints.Size;
+
+public class ServiceAddress {
+    @NotBlank(message = "Service Address defendantsAddress should not be empty")
+    @Size(max = 3, message = "Service Address defendantsAddress should not be longer than {max} characters")
+    private final String defendantsAddress;
```

Comment:
What is this field? It looks like it holds yes / no.

Reply:
it is supposed to be a yes/no but I didn't see a class in claim store to use but I was rushed getting it in before leaving yesterday evening

Reply:
boolean